docs/reference.rst

```diff
@@ -145,6 +145,10 @@ For more information about using casync support of RAUC, refer to
   By default, the chunk store path is derived from the location of the RAUC
   bundle you install.
 
+``tmpdir``
+  Allows to set the path to use as temporary directory for casync.
+  By default, the temporary directory is set to /var/tmp.
```

Comment:
You say that /var/tmp is used by default, but this is not configured explicitly when starting casync. It seems casync would use TMPDIR if set already, and then fall back to an internal casync default.
That seems to be a sensible behavior, but should be reflected in the docs.

Reply:
Done.

Hope my reword is good enough ;)